test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -550,6 +550,23 @@ describe('S2S Adapter', () => {
         }
       });
     });
+
+    it('converts appnexus params to underscore syntax', () => {
+      const s2sConfig = Object.assign({}, CONFIG, {
+        endpoint: 'https://prebid.adnxs.com/pbs/v1/openrtb2/auction'
+      });
+      config.setConfig({s2sConfig});
+
+      const myRequest = utils.deepClone(REQUEST);
+
+      debugger;//eslint-disable-line
```

Comment:
I always forget to remove this.

Reply:
Sorry about that - just pushed a commit to remove it.